app/models/reporting/table.rb

```diff
@@ -16,6 +26,17 @@ def rows
 
     private
 
-    attr_reader :table_hash
+    attr_reader :table_hash, :numeric_column_keys
+
+    def numeric_column?(key)
+      numeric_column_keys.include?(key)
+    end
+
+    # All bar the first column are assumed to be numeric
+    def default_numeric_column_keys
+      numeric_keys = table_hash.keys.dup
+      numeric_keys.shift
+      numeric_keys
```

Comment:
Is the `.shift` here because the first key is a header/title?

Reply:
Default behaviour is that all but first column is numeric.

Perhaps I should remove the default if confusing?